changes/xxxx.added

```diff
@@ -0,0 +1 @@
+Added `nautobot.apps.utils.get_url_for_url_pattern` and `nautobot.apps.utils.get_url_patterns` lookup functions.
```

Comment:
14 change log entries in 1 PR 😩

Reply:
Yeah 😢 This one scope crept, but I think everything in it is still relevant/necessary.